portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,15 +19,26 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays; 
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comment_list = new ArrayList<>(Arrays.asList("HE WAS NUMBER 1", 
+  "Wow. Guess what comment this is. Dos!", "Three! THREE! ThIrD yerrr"));;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Welcome to my page!");
+    //Convert comment_list into a JSON object.
+    Gson gson = new Gson();
+    String json = gson.toJson(comment_list);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
```

Comment:
I guess you will have to remove the ";" as a valid content type.

Reply:
The ";" in "application/json;"? Why  should I do that?

Reply:
Response content type itself has no `;`  in it. See a list of common types https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
So you should do `response.setContentType("application/json");`